src/gt4py/next/program_processors/runners/dace/gtir_domain.py

```diff
@@ -41,64 +39,23 @@ class FieldopDomainRange:
 """Domain of a field operator represented as a list of `FieldopDomainRange` for each dimension."""
 
 
-def extract_domain(node: gtir.Expr) -> FieldopDomain:
+def get_field_domain(domain: domain_utils.SymbolicDomain) -> FieldopDomain:
     """
     Visits the domain of a field operator and returns a list of dimensions and
     the corresponding lower and upper bounds. The returned lower bound is inclusive,
     the upper bound is exclusive: [lower_bound, upper_bound[
```

Comment:
```suggestion
    the upper bound is exclusive, i.e. `[lower_bound, upper_bound[`
```
I know I am bad ad spelling and grammar, but would this not be better?